recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -316,6 +316,11 @@ do_deploy() {
         echo "# Enable One-Wire Interface" >> $CONFIG
         echo "dtoverlay=w1-gpio" >> $CONFIG
     fi
+
+    # Reduce config.txt file size to avoid corruption and
+    # to boot successfully Raspberry Pi 5
+    sed -i '/^##/d' $CONFIG
```

Comment:
We ned to clean this up properly (at some point). Can we also add a reference to the upstream bug we created for this?

Reply:
Yes, I've just reported it: https://github.com/Evilpaul/RPi-config/issues/9

Reply:
I wish this was reported to the foundation as it is not a config text issue is a firmware limitation.

Reply:
Is there a way to get in touch with them and report it (apart from the forum)? @floion has already reported it through [the discussion in the Raspberry Pi forum](https://forums.raspberrypi.com/viewtopic.php?t=359838&sid=91ba6454119bc581013fa2f3fe922c60).

Reply:
I would create a firmware issue: https://github.com/raspberrypi/firmware

Reply:
@leon-anavi did you create an issue in the firmware repo? If yes, could you add it as a comment in the file so it's known the changes are temporary?

Reply:
Yes, I [reported the issue](https://github.com/raspberrypi/firmware/issues/1848) and I rebased the commit to include links in the comments for reducing `config.txt` size.

Reply:
Do we know why this started to happen out of nowhere only on rpi5?

Reply:
@agherzan, I am quoting the answer given to @leon-anavi [report](https://github.com/raspberrypi/firmware/issues/1848#issuecomment-1825868376) issue:

> Amongst other reasons the bounded size is that config.txt is now parsed earlier by the bootloader code which is has a bounded size for signature checks

Reply:
What about putting this lines in this code instead?

```
do_deploy:append:raspberrypi5() {
 ...
}
```

Note: Or if #1255 is merged, you may consider adding this `RPI_CONFIG_STRIP = "1"` in `raspberrypi5.conf`.

Reply:
@gportay Thanks - that makes sense now. I think we will address it in a separate MR.